### DISCUSSION TITLE
I have to start the GUI after reboot,

### DISCUSSION THREAD

**Discussion by Snekutaren** [evilsocket/opensnitch#520] [Q&A]
Everytime I reboot I have to restart the GUI to get notifications.
Is that how it works? I think I remember that I didn't have to at first.
Also the default action was suddently set to allow in default-config.json even though it was set to deny in GUI.
System service is enabled/started. Whats going on?

**Comment by gustavo-iniguez-goya** [comment]
Hi @Snekutaren ,

Please, tell us more about you(r system):
- OpenSnitch version
- Distribution
- Kernel version
- Desktop Environment / Window Manager
- output of `journalctl -ar|grep opensnitch-ui` after login in to your DE.
- Does opensnitch_ui.desktop exist under ~/.config/autostart/ ? it should be a symbolic link to /usr/share/applications/opensnitch_ui.desktop

> Also the default action was suddently set to allow in default-config.json even though it was set to deny in GUI.

One reason could be that you updated OpenSnitch and the package manager asked you to update daemon's configuration file or keep the one you had installed. If chose the first option it just installs the config file that is shipped with the package.

**Comment by Snekutaren** [comment]
Hi!
Thanks for replying and for the great program.
Here comes some info:

   - OpenSnitch version - *1.4.0-3*
   - Distribution - *Arch Linux x86_64*
   - Kernel version - *5.14.8-arch1-1 *
   - Desktop Environment / Window Manager - *GNOME 40.4 / Mutter*

*Does opensnitch_ui.desktop exist under ~/.config/autostart/ ? it should be
a symbolic link to /usr/share/applications/opensnitch_ui.desktop*

> It did not. Now the GUI starts at gnome startup. Is it possible to run it
> in the background somehow or do I have to start the GUI in foreground like
> this to have notifications?
> If I close the GUI after startup I still get notification popups.


*Output of journalctl -ar|grep opensnitch-ui after login in to your DE.*

> The command "journalctl -ar|grep opensnitch-ui" gives me
> "opensnitch-ui[11866]: Error reading events from display: Broken pipe"


*One reason could be that you updated OpenSnitch and the package manager
asked you to update daemon's configuration file or keep the one you had
installed. If chose the first option it just installs the config file that
is shipped with the package.*

> I did change from the git version to non-git..
> That's it. As you can tell I'm no Linux wiz. :)

**Reply by gustavo-iniguez-goya** [reply]
Thank you!

> It did not. Now the GUI starts at gnome startup. Is it possible to run it
 in the background somehow or do I have to start the GUI in foreground like
 this to have notifications?
 If I close the GUI after startup I still get notification popups.

ah ok. You need to install this extension https://extensions.gnome.org/extension/615/appindicator-support/ in order the icon to show up in the systray:
``` 
    Download latest version for your Gnome version - https://github.com/ubuntu/gnome-shell-extension-appindicator/releases
    Install it with your regular user: gnome-extensions install gnome-shell-extension-appindicator-v33.zip
```

By the way, if the simbolic link is not created under `~/.config/autostart/` or `/etc/xdg/autostart/` please, tell the AUR package mantainer about the problem, so they can solve it.
